Application Actions/Spotify/1.0/Increase Volume.scpt

```diff
@@ -0,0 +1,8 @@
+tell application "Spotify"
+	if (sound volume < 90) then
+		set the sound volume to (sound volume + 2)
+	else if (sound volume > 90) and (sound volume <= 100) then
```

Comment:
It should cover the case `sound volume >= 90` here as right now sound volume = 90 isn't covered.